nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -380,6 +389,10 @@ public boolean removeProperties(Namespace namespace, Set<String> properties) {
   }
 
   private boolean updateProperties(Namespace namespace, Consumer<Map<String, String>> action) {
+    if (namespace.isEmpty()) {
+      // As creating empty namespaces is not supported
+      throw new IllegalArgumentException("Does not support empty namespace");
```

Comment:
The sentence does not have a subject. How about "Empty namespaces not supported"?

Reply:
fixed.